gene_finder.py

```diff
@@ -45,8 +53,13 @@ def get_reverse_complement(dna):
     >>> get_reverse_complement("CCGCGTTCA")
     'TGAACGCGG'
     """
-    # TODO: implement this
-    pass
+    complement_list= []                          #creating empty complement list
```

Comment:
You don't need to comment every line - just comment the things that might be unclear to someone reading your code (or if you want to document something important).